### ISSUE TITLE
Blocks randomly enter an infinite loading state when they are added

### ISSUE THREAD

**Issue by TarikFojnica** [payloadcms/payload#10070]
### Describe the Bug

Blocks will randomly get stuck in the loading state after adding them to the layout. After I remove that block and add it again, it will probably work the next time, but will stop working randomly again if I keep adding.

It's not related to block quantity, sometimes stops at the first block. If one block is stuck, and I add the next one, they would resolve together.

https://github.com/user-attachments/assets/88bf428a-8830-4317-bf8e-da7186bf8984

### Link to the code that reproduces this issue

https://github.com/payloadcms/payload/tree/main/templates/website

### Reproduction Steps

The issue was reproduced in an official Website template. I uploaded the video that's demonstrating the issue. It was initially discovered on my custom project.

### Which area(s) are affected? (Select all that apply)

area: ui

### Environment Info

```text
payload: 3.7.0
node: 22.11.0
next: 15.1.0

The issue was also reproduced in payload: 3.9.0 on both Mac and Windows dev environment, as well as production environment. It can also be reproduced in Node 20.9.0
```

**Comment by jacobsfletch** [comment]
Thank you for reporting on this @TarikFojnica. Sounds like this could be the same issue as described in #8961. Looking into it.

**Comment by 1b0t** [comment]
This did come up for we as well today. Project is on 3.9.

**Comment by 1b0t** [comment]
This did come up for we as well today. Project is on 3.9.

**Comment by jacobsfletch** [comment]
Thanks @1b0t. After looking into this further, that issue is probably unrelated. I am also able recreate this on my end.

**Comment by leon-marzahn** [comment]
I have a decently complex page in payload and on the latest version i get this literally all the time
![image](https://github.com/user-attachments/assets/29e3d140-0a61-4e8d-867b-cbe08fd1c5fc)

Is there any timeline for the fix yet? @jacobsfletch

**Comment by jacobsfletch** [comment]
@leon-marzahn I've got a fix for this queued up in #10175.

**Comment by leon-marzahn** [comment]
@leon-marzahn Thanks for getting a fix out so quickly, when could we expect a release with this fix? I'm actually really struggling to work on a client website because i keep getting this infinite loading state
